README.md

```diff
@@ -138,6 +159,11 @@ The scripts perform (1) block pruning ➔ (2) LoRA-based retraining ➔ (3) zero
   bash script/evaluate.sh
   ```
 
+- (Optional) Any post-training quantization method can be applied on our pruned models. Following example script quantize pruned vicuna-7b model with GPTQ and measuring ppl & accuracy (same as `script/evaluate.sh`):
```

Comment:
Typos exist
- Following example script quantize -> “The” following … quantize”s”
- Would be hard to understand the meaning of “measuring ppl & accuracy” in the sentence

Reply:
How about changing the description into:

- Any post-training quantization method can be applied to our pruned models. The example script quantizes our pruned models using GPTQ and measures their performance with `script/evaluate.sh`:

Reply:
Thanks. I've incorporated this suggestions on README.